src/analysis/multiSpecies/microbiomeModelingToolbox/mgPipe/makeDummyModel.m

```diff
@@ -0,0 +1,50 @@
+function dummy = makeDummyModel(numMets, numRxns)
+% Makes an empty model with numMets rows for metabolites and numRxns columns
+% for reactions. Includes all fields that are necessary to join models.
+%
+% USAGE:
+%
+%   dummy = makeDummyModel(numMets, numRxns)
+%
+% INPUTS:
+%   numMets:       Number of metabolites
+%   numRxns:       Number of reactions
+%
+% OUTPUT:
+%   dummy:         Empty COBRA model structure
+%
+% .. Authors: SM June 2016
+
+dummy.mets = cell(numMets, 1);
+dummy.rxns = cell(numRxns, 1);
+dummy.rev = zeros(numRxns, 1);
+dummy.c = zeros(numRxns, 1);
+dummy.S = zeros(numMets, numRxns);
+dummy.lb = zeros(numRxns, 1);
+dummy.ub = zeros(numRxns, 1);
+dummy.b = zeros(numMets, 1);
+dummy.metNames = cell(numMets, 1);
+dummy.metFormulas = cell(numMets, 1);
+dummy.rxnNames = cell(numRxns, 1);
+dummy.subSystems = cell(numRxns, 1);
+dummy.genes = {};
+dummy.rxnGeneMat = zeros(numRxns, size(dummy.genes, 1));
+dummy.rxnGeneMat = sparse(dummy.rxnGeneMat);
+dummy.rules = cell(numRxns, 1);
+dummy.grRules = cell(numRxns, 1);
+dummy.comments = cell(numRxns, 1);
+dummy.citations = cell(numRxns, 1);
```

Comment:
`.comments` and `.citations` are undefined fields. 

Overall:
Wouldn't `createModel()` be better for your instance?
And then using `addReaction`/`addMetabolite` functions? 
That way you could be sure that the functions will also work in the future, while this manual setting of fields Might, or might not be supported in the future. 

There is currently a PR in which should help you quite a lot in allowing the batch addition of Metabolites/reactions, while keeping the model struct in sync.